### PR TITLE
Add logo check from stream

### PR DESCRIPTION
**Pull request by unanoc**
I decided to remove these values from config because it almost never changes, no need to store it in config it helps to make funcs cleaner.
I also added checking of logo size for logos which are uploaded from PR (stream)

### CODE REVIEW

File: pkg/validation/image.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import (
+	"fmt"
+	"image/png"
+	"io"
+	"os"
+)
+
+const (
+	kbInByte    = 1000
```
Comment (tw-covain) [inline pkg/validation/image.go]:
```suggestion
	bytesInKB    = 1024
```

File: pkg/validation/image.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+func ValidatePngImageDimension(file io.Reader) error {
+	img, err := png.DecodeConfig(file)
+	if err != nil {
+		return fmt.Errorf("failed to decode image: %w", err)
+	}
+
+	if img.Width > maxW || img.Height > maxH || img.Height < minH || img.Width < minW || img.Width != img.Height {
+		return fmt.Errorf("%w: logo should be 256x256: given %dx%d", ErrInvalidImgDimension, img.Width, img.Height)
+	}
```
Comment (tw-covain) [inline pkg/validation/image.go]: @vikmeup do we really support size 128x128? I guess it's not required

Comment (tw-covain) [reply pkg/validation/image.go]: Error says that logo should be 256x256, although it's flexible

Comment (unanoc) [reply pkg/validation/image.go]: I took this logic from the old bot, looks like ideally we need 256x256, but if it's less or bigger a bit - it's okay

Comment (vikmeup) [reply pkg/validation/image.go]: 256 by 256 is ideal or higher, but 128 min is fine for the tokens too.

but it make sense to pass size var into the error message for consistent, avoid hard coding numbers

**Pull request merged**